### ISSUE TITLE
Can't build default layout

### ISSUE THREAD

**Issue by purpleP** [jackhumbert/qmk_firmware#338]
```
../../quantum/quantum.c: In function ‘is_music_on’:
../../quantum/quantum.c:440:13: error: ‘music_activated’ undeclared (first use in this function)
     return (music_activated != 0);

```

You have
```

#ifdef AUDIO_ENABLE
  bool music_activated = false;
```
But I don't see this defined anywhere in ergodox folder. So I think that's the root of a problem. If it's not defined for ergodoxez then naturally there is no `music_activated` variable

**Comment by purpleP** [comment]
Okay, I've found this

```
quantum/quantum.mk
31:ifeq ($(strip $(AUDIO_ENABLE)), yes)
```
So I've tried setting AUDIO_ENABLE env variable and now it builds okay.

1. This should be stated in docs.
2. It still probably a bug. You either should have a default value for music_activated in case AUDIO_ENABLE is not defined or default value for AUDIO_ENABLE.
Actually I've just checked it wouldn't build if audio_enable=no
